txbits/app/service/sql/misc.scala

```diff
@@ -153,6 +153,13 @@ object misc {
     """.stripMargin
   )
 
+  val changeFeesToOneWay = SQL(
```

Comment:
Good point. I'll fix it. Either way, I feel like these sorts of sql statements don't belong in this file, but that's a job for another PR.